src/axom/inlet/Table.cpp

```diff
@@ -437,11 +442,8 @@ struct PrimitiveArrayHelper<Key, bool>
   PrimitiveArrayHelper(Table& table, Reader& reader, const std::string& lookupPath)
   {
     std::unordered_map<Key, bool> map;
-    if(!reader.getBoolMap(lookupPath, map))
-    {
-      SLIC_WARNING(
```

Comment:
See previous comment about whether this is still needed for required fields. Looks like there are a few of these. They are either all okay, or all need to be updated. I am not familiar enough with this code to know which one it is.